social_django/migrations/0002_add_related_name.py

```diff
@@ -12,10 +12,6 @@
 
 
 class Migration(migrations.Migration):
-    replaces = [
-        ('default', '0002_add_related_name'),
-        ('social_auth', '0002_add_related_name')
```

Comment:
This doesn't look right either. A squashed migration should not be replacing itself but instead it should be replacing others. Maybe that's the actual bug here - that the squash + replaces attributes were not implemented properly the first time around ?